Tests/ExtractionTests/ExtractionTests.swift

```diff
@@ -1,168 +1,195 @@
-        let expectedResults = try XCTUnwrap(extractionJob.expectedResults, "Snapshot at '\(snapshotURL.absoluteString)' does not contain expected results.")
-        let expectedResultsByProposalID = expectedResults.proposals.reduce(into: [:]) { $0[$1.id] = $1 }
-        
-        let extractedEvolutionMetadata = try await EvolutionMetadataExtractor.extractEvolutionMetadata(for: extractionJob)
-
-        // Check top-level properties
-        XCTAssertEqual(extractedEvolutionMetadata.commit, expectedResults.commit)
-        XCTAssertEqual(extractedEvolutionMetadata.creationDate, expectedResults.creationDate)
-        XCTAssertEqual(extractedEvolutionMetadata.implementationVersions, expectedResults.implementationVersions)
-        XCTAssertEqual(extractedEvolutionMetadata.schemaVersion, expectedResults.schemaVersion)
-        XCTAssertEqual(extractedEvolutionMetadata.toolVersion, expectedResults.toolVersion)
-        
         // Check proposals
-        for newProposal in extractedEvolutionMetadata.proposals {
-            let id = newProposal.id
-            guard !id.isEmpty  else {
-                continue
-            }
-            
-            guard let sourceProposal = expectedResultsByProposalID[id] else {
-                XCTFail("Unable to find id \(id) in expected results.")
-                continue
+        @Test func `Expected proposals`() async throws {
+            let expectedResults = try #require(extractionJob.expectedResults, "Snapshot at '\(snapshotURL.absoluteString)' does not contain expected results.")
+            let expectedResultsByProposalID = expectedResults.proposals.reduce(into: [:]) { $0[$1.id] = $1 }
+
+            for newProposal in extractedEvolutionMetadata.proposals {
+                let id = newProposal.id
+                guard !id.isEmpty  else {
+                    continue
+                }
+
+                guard let sourceProposal = expectedResultsByProposalID[id] else {
+                    Issue.record("Unable to find id \(id) in expected results.")
+                    continue
+                }
+
+                #expect(newProposal == sourceProposal)
             }
-            
-            XCTAssertEqual(newProposal, sourceProposal)
         }
-        
+
         // Check the generated JSON to catch issues such as removing properties from the schema
-        let expectedResultsURL = snapshotURL.appending(path: "expected-results.json")
-        let expectedJSONData = try Data(contentsOf: expectedResultsURL)
-        let actualJSONData = try extractedEvolutionMetadata.jsonRepresentation
-        XCTAssertEqual(expectedJSONData, actualJSONData)
-        
-        // Uncomment to write full JSON files out to compare in a diff tool
-        // try writeJSONFilesToPath(expected: expectedJSONData, actual: actualJSONData, path: "~/Desktop")
+        @Test func `Serialization`() throws {
```

Comment:
Since the raw identifier is only a single word, the test shows up as 'Serialized()' in reports with the parenthesis.

If I understand correctly, could either add a test description to @Test or add another word to the method name.

Maybe 'Expected Serialization`